src/js/HScrollMenu.js

```diff
@@ -4,11 +4,10 @@ import MenuFooter from './MenuFooter';
 import store from './store.js'
 import UIController from './Controllers/UIController';
 import { connect } from 'react-redux'
-import { updateResetPeriod, resetTimeout } from './actions';
-import { DEFAULT_RESET_TIMEOUT } from "./Alert"
+import { resetTimeout } from './actions';
 
 const OUT_OF_BOUND_RESET_PERIOD = 1000;
-class HScrollMenu extends React.Component {
+export default class HScrollMenu extends React.Component {
```

Comment:
@LiliiaShlikhtLuxoft Why `export default` ?

Reply:
@ValeriiMalkov because every class should be exported, but previous commit changed it and put it at the end of class (for props purpose). I returned it.